packages/transpiler/src/transformations/fix-import-directives.ts

```diff
@@ -0,0 +1,20 @@
+import { getImportDirectives, getSourceIndices } from '../solc/ast-utils';
+import { Artifact } from '../solc/artifact';
+import { Transformation } from '../transformation';
+
+export function fixImportDirectives(artifact: Artifact, artifacts: Artifact[], contracts: string[]): Transformation[] {
+  const imports = getImportDirectives(artifact.ast);
+  return imports.map(imp => {
+    const [start, len] = getSourceIndices(imp);
+    const isTranspiled = artifacts.some(
+      art => art.ast.id === imp.sourceUnit && contracts.some(contract => contract === art.contractName),
+    );
+    const prefix = !imp.file.startsWith('.') ? './' : '';
```

Comment:
Since all the transpilied contracts are inside `__upgradeable__` they don't need it in their path because if they import files inside `__upgradeable__` they are fine with relative paths. If it is absolute reference that is also fine. The only case it doesn't cover right now if upgraded contract reference a not upgraded contract from user `contracts` folder. That is related to the `import` issue above. Let me think about that.
![image](https://user-images.githubusercontent.com/3285000/72526062-84134680-38a0-11ea-904e-6224a3964a25.png)